src/__tests__/modules/RegisterFormModule/RegisterModule.test.tsx

```
@@ -1,13 +1,326 @@
+    render(<RegisterModule />);
+    
+    // Fill form data without files
+    await userEvent.type(screen.getByPlaceholderText("Nama Depan"), "John");
+    await userEvent.type(screen.getByPlaceholderText("Nama Belakang"), "Doe");
+    await userEvent.type(screen.getByPlaceholderText("Masukkan email Anda"), "john@example.com");
+    await userEvent.type(screen.getByPlaceholderText("Masukkan nomor WhatsApp Anda"), "081234567890");
+    await userEvent.type(screen.getByPlaceholderText("Masukkan NIK Anda"), "1234567890123456");
+    await userEvent.type(screen.getByPlaceholderText("Masukkan NPWP Anda"), "123456789012345");
+
+    // Mock the validation function
+    const mockValidateStepOneForm = jest.spyOn(require('@/lib/validation/stepOneFormValidation'), 'validateStepOneForm');
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)